src/ApparentHorizons/Tags.hpp

```diff
@@ -366,6 +366,23 @@ struct RicciScalarCompute : RicciScalar, db::ComputeTag {
   using return_type = Scalar<DataVector>;
 };
 
+/// The pointwise maximum of the Strahlkorper's intrinsic Ricci scalar
+/// curvature.
+struct MaxRicciScalar : db::SimpleTag {
+  using type = double;
+};
+
+/// Computes the pointwise maximum of the Strahlkorper's intrinsic Ricci
+/// scalar curvature.
+struct MaxRicciScalarCompute : MaxRicciScalar, db::ComputeTag {
+  using base = MaxRicciScalar;
+  using return_type = double;
+  static double function(const Scalar<DataVector>& ricci_scalar) {
```

Comment:
@kidder Do you require compute tags return by reference so soon that this one should be updated now, or should we merge as-is and change later?

Reply:
@MarloMo In case @kidder wants you to change this now, all you need to do is this:

```cpp
static void function(const gsl::not_null<double*> max_ricci_scalar, const Scalar<DataVector>& ricci_scalar) noexcept {
  *max_ricci_scalar = max(get(ricci_scalar));
}
```

You might as well just do that now, and squash the change right away.

Reply:
yes, please do this